src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -187,6 +188,17 @@ default String ignoredNames() {
         return "";
     }
 
+    @ConfigItem(
+        keyName = "playerLookupService",
+        name = "Player lookup service",
+        description = "The service used to lookup a players name, in Discord embeds",
```

Comment:
```suggestion
        name = "Player Lookup Service",
        description = "The service used to lookup a players account, to make their name clickable in Discord embeds",
```
@Felanbird ?

Reply:
works for me, couldn't think of anything good while napping